### PR TITLE
Add id.sideka.App

### PR DESCRIPTION
**Pull request by uajain**
Village management application for Indonesia - http://sideka.id/

### CODE REVIEW

File: id.sideka.App.appdata.xml [hunk 1]
```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- See https://wiki.gnome.org/GnomeGoals/AppDataGnomeSoftware -->
+<component type="desktop">
+  <id>id.sideka.App.desktop</id>
+  <metadata_licence>CC0</metadata_licence>
+  <project_license>AGPL</project_license>
```
Comment (TingPing) [inline id.sideka.App.appdata.xml]: Invalid license; https://spdx.org/licenses/

**Comment by TingPing** [comment]
bot, build id.sideka.App

**Comment by nedrichards** [comment]
Ivalid JSON https://flathub.org/builds/#/builders/2/builds/1147

**Comment by uajain** [comment]
Fixed formatting issues: PR updated. Please trigger rebuild.

**Comment by nedrichards** [comment]
bot, build id.sideka.App

**Comment by nedrichards** [comment]
If you want aarch64 you have to use electron 1.8.x there isn't a build for 64 bit arm on 1.7.x:

https://github.com/electron/electron/releases/tag/v1.8.6

**Comment by uajain** [comment]
Upgrading electron to 1.8.x is not ideal I think as it has issues with app-startup. Therefore I would stick to 1.7.x and will add a flathub.json for "only_arches"

**Comment by nedrichards** [comment]
bot, build id.sideka.App

**Comment by nedrichards** [comment]
Repository has been created: https://github.com/flathub/id.sideka.App

**Comment by nedrichards** [comment]
@uajain repo created. is there anyone from upstream who we should add as co-maintainers?

**Comment by uajain** [comment]
@nedrichards hmm, not sure. It would be great if someone from upstream signs up for maintaining it but I have not communicated with any of upstream people. One way, is to open an issue of sideka repo and ask them about this?

**Comment by nedrichards** [comment]
@uajain how about a PR that contributes the desktop and appdata files? That's always a good way to introduce things.

**Comment by uajain** [comment]
Ok, I will submit a PR with desktop and appdata to the Sideka upstream repo

**Pull request closed without merging**